<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3500</width>
    <height>4200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>System Setting</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>ADwin System</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CPU Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Analog Out
Channel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Digital Out
Channel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="Cpu_type">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>25</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>T9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T12</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Analog_channel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Close</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Digital_channel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>95</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Close</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>64</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>96</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Control Program</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Offline Mode</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Offline_mode">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Enable the program to run simulated run (like outputting files),&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;without connecting to an ADwin system.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>105</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TCP Remote</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="TCP_switch">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>105</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="TCP_Port">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Port No.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pre-calculate cycle</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Precalculation">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>52</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>75</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stop after Scan</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="StopafterScan">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>78</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Apply">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>510</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Profiles</string>
   </property>
   <widget class="QTableWidget" name="File_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>401</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Path</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Comment</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Run</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Scan</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="File_import">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QPushButton" name="File_load">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Files Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="File_remove">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Move_up">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Move Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Move_down">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Move Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mission_start">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mission
Start</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
